examples/mapping/mi-mapping/src/main/wso2mi/artifacts/apis/Mapping.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/mapping" name="Mapping" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="POST" uri-template="/">
+        <inSequence>
+            <bal_mapping.mapPatient>
```

Comment:
Have we explained anywhere in docs, how `bal_mapping.mapPatient` is derived? if not we have to do that.

Reply:
This is added through UI in the VsCode which shows the available functions in the module. Added a description explaining the exposing of public functions as well.